src/openrct2/object/LargeSceneryObject.cpp

```diff
@@ -67,6 +67,15 @@ void LargeSceneryObject::ReadLegacy(IReadObjectContext* context, OpenRCT2::IStre
             context->LogError(ObjectError::InvalidProperty, "Sell price can not be more than buy price.");
         }
     }
+
+    // RCT2 would always remap primary and secondary colours for large scenery
+    // This meant some custom large scenery objects did not get exported with the required flags, because they still
+    // functioned, but without the ability to change the colours when the object was selected in the scenery window.
+    // OpenRCT2 changes the rendering so that the flags are required, we therefore have to assume all custom objects
+    // can be recoloured. The minor drawback to this, is that some custom large scenery will have the option to change
+    // the primary and secondary colour, however no effect will be seen.
```

Comment:
I think we should address this too. Perhaps have flags to suppress the colour picker for colours 1 and 2.

If there is a _rush_ to this PR in, it may be done later.

Reply:
@Gymnasiast I think it is better that some objects which incorrectly don't have the picker are fixed than objects which correctly disable the picker now have them.